### PR TITLE
[PREVIEW] - PDF annotation with event payload

### PR DESCRIPTION
**Pull request by smogese**

### CODE REVIEW

File: api/index.js [hunk 1]
```diff
@@ -1,47 +1,47 @@
...
+hearingRoutes(router)
+caseRoutes(router)
+decisionRoutes(router)
+emAnnoApiRoutes(router)
+emNpaApiRoutes(router)
 
 // Uncomment to enable direct access to Microservices
 // barApiRoutes(router);
-// ccdDefApiRoutes(router);
+ccdDefApiRoutes(router)
```
Comment (alectronic0) [inline api/index.js]: please comment back out please

Comment (smogese) [reply api/index.js]: Done

**Comment by smogese** [comment]
Sing quotes as opposed to " double quotes are the standard for string definitions in javascript followed by the vast majority of projects/frameworks. Although like most standards not always taken up. It is the recommended standard in eslint for javascript.

File: api/auth/auth.spec.js [hunk 2]
```diff
@@ -1,56 +1,59 @@
...
+
+    it('Should redirect to /', () =>
+        request
+            .get('/oauth2/callback')
+            .expect(302)
+            .then(res => {
+                expect(res.headers.location).toEqual('/')
+            }))
+
+    // it('Should convert the idam code toa jwt', () =>
```
Comment (alectronic0) [inline api/auth/auth.spec.js]: could please fixes these rather than commenting out please

**Pull request merged**